src/fabric-video-expandable/web/index.js

```diff
@@ -24,7 +24,7 @@ function handleToggle() {
 }
 
 function onToggle(isOpen, toggle, target) {
-    resizeIframeHeight((isOpen ? 500 : 250) + (showLabel === 'yep' ? labelHeight : 0));
+    resizeIframeHeight((isOpen ? 500 : 250) + (showLabel === 'yes' ? labelHeight : 0));
```

Comment:
Though - does this now require the CSS classes to be changed? Looks like there are a few classes with `-$showLabel`